docker-compose.local.yml

```
@@ -29,7 +28,7 @@ services:
       - vim-cms-data:/var/lib/mysql
       - ./db/docker-scripts:/docker-entrypoint-initdb.d
     ports:
-      - 3306:3306
+      - ${DATABASE_PORT}:3306
```

here as well with the port:

```suggestion
      - ${DATABASE_PORT}:${DATABASE_PORT}
```